invenio_rdm_records/requests/community_submission.py

```diff
@@ -18,6 +18,21 @@
 #
 # Actions
 #
+class CommentAction(RequestAction):
```

Comment:
you don't need to create the class for Comment action, it is already implemented but not explicitly as a separate class. Check the requests service, it basically needs only using the create method with a specific type in the payload
```
"type": RequestEventType.COMMENT.value
```